tests/test_processing_common.py

```diff
@@ -44,6 +44,13 @@
     from transformers import CLIPImageProcessor
 
 
+def prepare_image_inputs():
```

Comment:
Move out as a function s.t. processing classes which can't use `ProcessorTesterMixin` in their test module can still import and re-use this logic instead of reimplementing